src/Platforms/MySQL80Platform.php

```diff
@@ -9,6 +9,16 @@
  */
 class MySQL80Platform extends MySQL57Platform
 {
+    /**
+     * {@inheritdoc}
+     *
+     * @link https://dev.mysql.com/doc/refman/8.0/en/date-and-time-literals.html#date-and-time-string-numeric-literals
+     */
+    public function getDateTimeTzFormatString()
```

Comment:
Overriding `getDateTimeTzFormatString` without overriding `getDateTimeTzTypeDeclarationSQL` (so that it does not fallback to the non-tz type) will not provide a working implementation of the type (it will actually make things worse than doing the silent fallback consistently)

Reply:
I'm afraid I'm missing the point here.
AFAIK, there are no specific types on MySQL for `TIMESTAMP` nor `DATETIME`:
>MySQL converts TIMESTAMP values from the current time zone to UTC for storage, and back from UTC to the current time zone for retrieval. (This does not occur for other types such as DATETIME).

Reply:
The point of the tz type is that it stores a value for a precise datetime, ensuring that when reading it again, we get the same point in time without having to agree on which timezone is implicitly used by the code.

For a tz type, we need to make sure that on insertion, we can provide a timezone and it is not ignored, but also that on reads, we can get a timezone info from the DB.
If MySQL works only with its implicit timezone on reads, it means we cannot support a tz type properly in DBAL (note that I'm not a fan of the fact that DBAL silently falls back to the non-tz type as it hides the fact that your db is not timezone aware anymore).

There is a good reason is the [EcmaScript proposal for the Temporal API](https://tc39.es/proposal-temporal/docs/) has separate objects representing ZonedDateTime and PlainDateTime.